ActivityListener.Tests/Dockerfile

```diff
@@ -6,8 +6,19 @@ ENV DOTNET_CLI_TELEMETRY_OPTOUT='true'
 ENV DynamoDb_LocalMode='true'
 ARG LBHPACKAGESTOKEN
 ENV LBHPACKAGESTOKEN=$LBHPACKAGESTOKEN
+
+ARG SONAR_TOKEN
+ENV SONAR_TOKEN=$SONAR_TOKEN
+
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y openjdk-11-jdk
+RUN dotnet tool install --global dotnet-sonarscanner
+ENV PATH="$PATH:/root/.dotnet/tools"
+
+RUN dotnet sonarscanner begin /k:"LBHackney-IT_activity-listener" /o:"lbhackney-it" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"
```

Comment:
Please double check if the project key will be activity-listener or activity-history-listener

Reply:
I've checked and the project key is `activity-listener`

Reply:
The project was not imported. After I imported it, the project key is LBHackney-IT_activity-history-listener